RecoLocalMuon/GEMRecHit/src/ME0RecHitBaseAlgo.cc

```diff
@@ -33,12 +33,9 @@ const ME0DigiPreRecoCollection::Range& digiRange){
     // Call the compute method
     bool OK = this->compute(*digi, point, tmpErr);
     if (!OK) continue;
-   
-    if (std::abs(digi->pdgid()) == 13) {
-       ME0RecHit* recHit = new ME0RecHit(me0Id,digi->tof(),point,tmpErr);
-       result.push_back(recHit);
-    }
-
+    if (recOnlyMuons && std::abs(digi->pdgid()) != 13)  continue;
```

Comment:
Hi Slava

We prefer to fold in MC -truth for now, to make the performance studies. I ll present our ideas in tomorrows RECO meeting. I understand there are good reasons not to include it, but I would like to discuss this in person.
For sure we will remove the MC truth once necessary studies are performed. Anyhow 30th of Nov should be reasonable.